docs/src/start/getting-started.md

```diff
@@ -107,40 +120,39 @@ To connect the device to the Cumulocity IoT it needs to be configured.
 
 This URL is needed to allow the upload of the certificate to the specific tenant and the registration of the device. It can be configured via:
 
+<UserContext>
+
 ```sh
-sudo tedge config set c8y.url "${YOUR_C8Y_URL}"
+sudo tedge config set c8y.url "$C8Y_URL"
 ```
 
-```sh title="Example"
-sudo tedge config set c8y.url "mycompany.cumulocity.com"
-```
+</UserContext>
 
 ### Certificate
 
 %%te%% connects via MQTT protocol using a X.509 certificate for authentication. To do so, a certificate must be trusted by Cumulocity IoT. A certificate is trusted when it is added to the trusted certificates and is in an activated state.
 
 First, we need to create the device certificate locally (If the device certificate is already uploaded, directly via the UI to Cumulocity IoT this step can be skipped).
 
+<UserContext>
+
 ```sh
-sudo tedge cert create --device-id "${DEVICE_ID}"
+sudo tedge cert create --device-id "$DEVICE_ID"
 ```
 
-```sh title="Example"
-export DEVICE_ID="gateway_4932C0CE7D12"
-sudo tedge cert create --device-id "${DEVICE_ID}"
-```
+</UserContext>
 
 The device id is a unique identifier e.g. the MAC address that identifies the physical device.
 
 The certificate is uploaded to the Cumulocity IoT Tenant via:
 
+<UserContext>
+
 ```sh
-sudo tedge cert upload c8y --user "${C8Y_USER}"
+sudo tedge cert upload c8y --user "$C8Y_USER"
 ```
 
-```sh title="Example"
-sudo tedge cert upload c8y --user "john.smith@example.com"
-```
```

Comment:
Good to get rid of these examples.

Reply:
Good catch. I had removed most of the other examples but missed this one.